docs: add `sips-preview.yazi` to preloaders section

andreas-timm opened the pull request:
[sips-preview.yazi](https://github.com/andreas-timm/sips-preview.yazi) -
  Quick previews of images on macOS. It focuses on PSD (Photoshop) files but also supports other image formats, utilizing the sips command-line tool.

sxyazi commented:
Thank you for creating this plugin, it looks awesome!

Here are a few suggestions for the README:

- Using `prepend_preloaders` is better than `preloaders`, as the latter will override all other preloaders.
- It seems you forgot to add `prepend_previewers`, or maybe I misunderstood the purpose of this plugin.
- You can use [`ya pack`](https://yazi-rs.github.io/docs/cli#package-manager) instead of `git clone` to make it easier for users to install the plugin and always keep it up to date.

sxyazi reviewed (approved):
Thanks!

andreas-timm commented:
Thank you!

> `prepend_preloaders`, `prepend_previewers`

Fixed, corrected.. I stabilised the first `seek` call, previously the preview was not always displayed, for this I changed the logic used in the code preset plugins ℹ️

> `ya pack`

Yes, of course, added.
